sklearn/tests/test_multioutput.py

```diff
@@ -336,3 +341,123 @@ def test_multi_output_exceptions():
     y_new = np.column_stack((y1, y2))
     moc.fit(X, y)
     assert_raises(ValueError, moc.score, X, y_new)
+
+
+def test_classifier_chain_fit_and_predict():
+    """Fit classifier chain and verify predict performance"""
+    X, Y = datasets.make_multilabel_classification(n_samples=10000,
```

Comment:
That may be a bug in `cross_val_predict`. However:
* it may be appropriate to construct or use a multilabel dataset in another way, particularly if our validation that it is working goes beyond "jaccard > 0.5?" which ideally it should
* I don't see why you need `n_classes=10`. Surely this proves nothing more than `n_classes=3`
* `n_labels` controls the density of labels. Increasing it will mean you will very likely have positive examples for smaller `n_samples`.